tests/generators/kzg_4844/main.py

```diff
@@ -384,6 +424,38 @@ def case05_verify_blob_kzg_proof():
         'output': False
     }
 
+    # Correct `G1_POINT_AT_INFINITY` proof and commitment for zero poly
+    blob = BLOB_ALL_ZEROS
```

Comment:
Just note that this test case is duplicated to the first test case in `verify_blob_kzg_proof_case_correct_proof_*` test cases. But I think it's good to make it explicit.

Reply:
Yep, indeed. I also considered it being explicit a positive thing.